translations/fr.json

```diff
@@ -12,9 +12,8 @@
   "news.title": "NOUVELLES DE POLYTOPIA",
   "news.description": "Bienvenue !\n C'est ici que nous partageons les dernières nouvelles de l'univers de Polytopia. Restez connectés !",
   "gamemodepicker.title": "MODE DE JEU",
-
```

Comment:
Why delete spaces which help the reading?

Reply:
I have no control over this - I just pass forward propositions that I get through the website form. But in general, new lines are ignored in the JSON format, so if you use some tool to edit JSON file you will probably lose them, as they have no semantic meaning.

About the rest of the changes - these are not my changes, please incorporate them into your translation when they make sense, ignore the rest. If I have contact info to the author I will invite him/her to the discussion, but they did not leave any.

Reply:
Hello, i'm so sorry because when I leave comments this morning, I would like to let a review comment instead of add new random comment... So the first wich appears is a random comment.... At the end of this long review, I propose a requested changes to EZ363 if he want. 
For the other request, It was plenty of obvious mistakes... i let some comments and rejected the request. I think this one could be merge maybe. I already have commit the good suggestions and had the new editors.